src/server/pps/server/api_server.go

```diff
@@ -690,33 +723,16 @@ func (a *apiServer) ListJobSet(request *pps.ListJobSetRequest, serv pps.API_List
 				return nil
 			}
 		}
-		jobInfos, err := pachClient.InspectJobSet(id, request.Details)
+		jobset, err := getJobSet(jobInfo)
 		if err != nil {
-			return err
+			return errors.Wrap(err, "error filtering jobset")
```

Comment:
Is the error in _filtering_ or in _getting_ the job set?

As noted above, if `getJobSet` were expanded in place then each error return could give some more context.